x-pack/plugins/exploratory_view/public/components/shared/filter_value_label/filter_value_label.tsx

```diff
@@ -76,8 +77,8 @@ export function FilterValueLabel({
   const filter = buildFilterLabel({ field, value, label, dataView, negate });
 
   const {
-    services: { uiSettings },
-  } = useKibana();
+    services: { uiSettings, docLinks },
+  } = useKibana<ObservabilityAppServices>();
```

Comment:
Uptime team, could you please please test these changes with your data set? 
Btw, is it expected that components are duplicated under `exploratory_view` and `observability` plugins?

Reply:
Hey there @jughosta! We just merged a PR last week that cleans up these duplicates (https://github.com/elastic/kibana/pull/157970). Could you pull in main? If there's more duplicates please file an issue with us (Actionable Observability).